src/sentry/release_health/release_monitor/base.py

```diff
@@ -0,0 +1,17 @@
+from typing import Mapping, Sequence
+
+from sentry.utils.services import Service
+
+
+class BaseReleaseMonitorBackend(Service):
+    CHUNK_SIZE = 1000
+    MAX_SECONDS = 60
+
+    __all__ = ("fetch_projects_with_recent_sessions",)
+
+    def fetch_projects_with_recent_sessions(self) -> Mapping[int, Sequence[int]]:
+        """
+        Fetches all projects that have had session data in the last 6 hours, grouped by
```

Comment:
Maybe this should be `release health data` instead of `session`. @ahmedetefy How do we actually refer to sessions when we fetch them from metrics?